<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreInfoChangeWidget</class>
 <widget class="QWidget" name="StoreInfoChangeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改</string>
  </property>
  <widget class="QLabel" name="StoreName_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>店名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Leader_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>负责人：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Tel_label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>电话：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Operation_label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Turnover_label">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>营业额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="StoreName_lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Address_lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Leader_lineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Tel_lineEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Operation_lineEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Turnover_lineEdit">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>80</y>
     <width>187</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="Confirm_pushButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel_pushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
